mlir/test/Target/LLVMIR/genx.mlir

```diff
@@ -1,33 +1,33 @@
 // RUN: mlir-translate -mlir-to-llvmir -split-input-file %s | FileCheck %s
 
-llvm.func @genx_special_regs() -> i64 {
+llvm.func @genx_special_regs() -> i32 {
   // CHECK-LABEL: genx_special_regs
-  // CHECK: call i64 @_Z12get_local_idj(i32 0)
-  %1 = genx.workitem.id.x : i64
+  // CHECK: [[CI:%.*]] = call i64 @_Z12get_local_idj(i32 0)
+  // CHECK-NEXT: trunc i64 [[CI]] to i32
+  %1 = genx.workitem.id.x : i32
```

Comment:
Can we also keep some tests that return i64 ? Maybe one per operation kind ?

Reply:
changed all y back to i64